data/compassql_examples/input/3d-NxOxQ.json

```diff
@@ -5,11 +5,11 @@
     {
       "channel": "?",
       "field": "Cylinders",
-      "type": "nominal"
+      "type": "ordinal"
```

Comment:
Why change the input?

Reply:
these are all flip-flopped (cylinder should be ordinal, origin should be nominal)

Reply:
Were they wrong in the original queries?